R/pkg/R/mllib.R

```diff
@@ -661,7 +665,10 @@ setMethod("fitted", signature(object = "KMeansModel"),
 #  Get the summary of a k-means model
 
 #' @param object a fitted k-means model.
-#' @return \code{summary} returns the model's features, coefficients, k, size and cluster.
+#' @return \code{summary} returns summary information of the fitted model, which is a list.
+#'         The list includes the model's \code{coefficients} (model cluster centers),
```

Comment:
For the return list, I didn't see features and k.  Does R function not only return the last line?